admin_manual/installation/system_requirements.rst

```diff
@@ -12,31 +12,33 @@ Nextcloud needs a minimum of 128MB RAM, and we recommend a minimum of 512MB.
 Recommended setup for running Nextcloud
 ---------------------------------------
 
-For best performance, stability, support, and full functionality we recommend:
+For best performance, stability, and functionality we recommend:
 
 * Red Hat Enterprise Linux 7 / Ubuntu 16.04 LTS
 * MySQL/MariaDB
 * PHP 7.0 +
```

Comment:
Active support for 7.0 will end with the end of this year. Please change this to 7.1 + (active EOL end of 2018) 😉

Reply:
The current Ubuntu LTS still uses php 7.0 and will provide fixes until 2021. Nextcloud itself drops PHP support for version 5.6 with NC 14, as far as I know the support for php 7.0 is not dropped...

PHP 7.0 won't be supported beyond 2018, however many distributions provide upgrades and we could keep that at least until the end of 2018. You risk that people think that php 7.0 does not work with NC any more and they try use 3rd-party packages in their distributions even though php 7.0 would be perfectly fine.

Reply:
Indeed